source/assets/sass/visual-design/_typography.scss

```diff
@@ -210,7 +187,7 @@ pre {
 
 code {
   // Scale the code font size down, because the font itself is much larger.
-  font-size: math.div($sl-font-size--small, $sl-font-size--medium) * 1em;
+  font-size: 0.88889em;
```

Comment:
I wasn't sure if showing the math was important here as it seems like it is arbitrary to get the type to be a bit smaller

Reply:
Yeah, I wonder if both code and caps can use a shared 'slightly smaller' font size (I don't know what to actually call it)?

Reply:
I added a new size called sl-font-size--smaller with a value of 90% and it matches pretty close. That way we could add it to other places in the future and the name is generic enough, but explains what is happening. Thoughts @mirisuzanne?